content/rs/release-notes/rs-5-6-0-april-2020.md

```diff
@@ -0,0 +1,158 @@
+```
+
+For more information, check out the [Redis Enterprise Software installer documentation]({{< relref "/rs/installing-upgrading/_index.md" >}}).
+
+### Hyperloglog on Active-Active
+
+RS 5.6.0 adds the support of [HyperLogLog](http://www.antirez.com/news/75) data structure for [Active-Active Redis]({{< relref "/rs/administering/active-active.md" >}}) Databases.
+
+Because HyperLogLog is a counting data structure by nature, conflicts can occur when deleting entries. For efficiency, performance and memory considerations, conflicts between instances are resolved with DEL (delete) operations winning over ADD operations that took place in concurrent or before the DEL operation.
+
+For more information, check out the [HyperLogLog on Active-Active documentation]({{< relref "/rs/developing/crdbs/developing-hll-crdb.md" >}}).
+
+### RedisJSON on Redis on Flash
+
+RS 5.6.0 adds support for [Redis on Flash]({{< relref "/rs/concepts/memory-architecture/redis-flash.md" >}}) (RoF) databases with [Redis Modules]({{< relref "/modules/_index.md" >}}). [RedisJSON]({{< relref "/modules/redisjson/_index.md" >}}) is the first module to be available to run on a Redis on Flash database.
+
+Using RedisJSON on Redis on Flash allows you to benefit from high performance redis with high data volume at lower costs.
+
+### OSS Cluster API Support
+
+**Support for Active-Active and Replica Of databases**
+
+You can configure [Active-Active]({{< relref "/rs/administering/active-active.md" >}}) and [Replica Of]({{< relref "/rs/administering/active-passive.md" >}}) databases to use the [OSS Cluster API]({{< relref "/rs/concepts/data-access/oss-cluster-api.md" >}}) using the web UI.
+The OSS Cluster API improves the performance of user operations against your database.
+
+You can also create or modify an Active-Active Redis database in OSS Cluster mode using the `crdb-cli` tool with the `--oss-cluster` option to apply the changes to all of the instances.
+
+For more information, check out the [OSS Cluster API documentation]({{< relref "/rs/administering/designing-production/networking/using-oss-cluster-api.md" >}}).
+
+**Create and edit database using the web UI**
+
+You can configure OSS Cluster API for databases using the web UI.
+
+For Active-Active databases, you can create the database with OSS Cluster API enabled for all of its instances.
+When you enable OSS Cluster after the Active-Active database is created, the change applies only to the local instance.
+
+### Slave HA defaults for Active-Active
+
+To enhance the availability and the consistency of [Active-Active Geo-Distributed Redis]({{< relref "/rs/administering/active-active.md" >}}),
+[Slave HA]({{< relref "/rs/administering/database-operations/slave-ha.md" >}}) is enabled by default for all existing and new Active-Active Redis Databases.
+
+To disable Slave HA for for a local instance of an Active-Active database, run this command on the instance: `rladmin tune db <bdb_uid> slave_ha disabled`
+
```

Comment:
You removed the note intentionally? 

> Note that slave HA follows all prerequisites of slave migration. In order to start slave shards that reside on a failed node, on new node/s, the new node/s should have enough DRAM to hold these shards. Otherwise, the slave shards cannot be started.

Reply:
Yes because I added this information into the text.